### PR TITLE
Update to f0b9f99

### PR DESCRIPTION
**Pull request by brianjmurrell**
To be in sync with cart.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -40,215 +40,7 @@
...
-                    }
-                }
-            }
-        }
-    }
-}
+// no openpa for leap 15 yet
+// ubuntu deps need to catch up
+packageBuildingPipeline(['distros' : ['centos7', 'sles12.3', 'leap42.3',
+                                     'leap15']])
```
Comment (JohnMalmberg) [inline Jenkinsfile]: No newline at EOF

File: Makefile [hunk 2]
```diff
@@ -1,29 +1,14 @@
...
-else
-PATCHES := c68870ffc0409c29eece5ba036c6efd3c22cee41^...v1.0.1.patch
-endif
+PATCHES := v1.0.1...cc0807e8377e129945834d292be21a6667a8cbb3.patch                                   \
+	       cc0807e8377e129945834d292be21a6667a8cbb3...f0b9f992793be46f1c6ae47b30d1c3ccb525cfbf.patch
 
-c68870ffc0409c29eece5ba036c6efd3c22cee41^...v1.0.1.patch:
-	curl -f -L -O '$(PATCH1)'
+el7_PR_REPOS  := libpsm2@PR-1
+PR_REPOS      := libfabric@PR-16
```
Comment (JohnMalmberg) [inline Makefile]: Don't we want to get these PRs landed first?

Comment (brianjmurrell) [reply Makefile]: Oh, good catch.  Those shouldn't be in there.

Comment (brianjmurrell) [reply Makefile]: Fixed.  Those updates for PSM2 will be in the next round.

**Pull request merged**